python/test/result_test.py

```diff
@@ -113,7 +113,7 @@ def test_to_score_str(self):
         self.result.set_score_aggregate_method(ListStats.total_variation)
         self.assertAlmostEquals(self.result.get_result('VMAF_legacy_score'), 6.5901873052628375, places=4)
         self.result.set_score_aggregate_method(partial(ListStats.moving_average, n=2))
-        self.assertItemsEqual(self.result.get_result('VMAF_legacy_score'),
+        self.assertEqual(list(self.result.get_result('VMAF_legacy_score')),
```

Comment:
A few of those `assertSomething` methods were removed in python3's unittest...

The consensus is roughly that there are too many of them, they're a lot java-like, and don't really provide much value...

With a modern test runner (like `pytest`), you don't need this at all.
A simple `assert a == b` will usually be sufficient (you will get a diff output of the difference too, no need for elaborate functions)